corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js

```diff
@@ -46,12 +46,30 @@ hqDefine("cloudcare/js/formplayer/menus/views/query", function () {
             'change': 'render',
         },
 
+        onBeforeRender: function() {
+            // Remove focus from any selectWoos that already exist because
+            // they will be replaced in onRender. When overwritten selectWoos
+            // think they have focus, they can respond to keypresses and
+            // spontaneously pop up and hide part of the HQ container.
+            if (this.selectWoos) {
```

Comment:
Would it make sense to just iterate over the `ui.valueDropdown` elements, since we know that they'll all be selectWoos? It'd be one less variable to keep track of.

Reply:
Yes, I tested this out in an experiment that I abandoned, and it is better.